### ISSUE TITLE
with_sequence gives a random order

### ISSUE THREAD

**Issue by tharanga-abeyseela** [ansible/ansible#30798]
##### ISSUE TYPE

 - Bug Report
 

##### COMPONENT NAME

  -  with_sequence
##### ANSIBLE VERSION

```
ansible-playbook 2.3.2.0
```

##### OS / ENVIRONMENT

3.10.0-514.6.2.el7.x86_64
##### SUMMARY

This is my playbook.

---
- hosts: all
  tasks:
  - name: sequence check
    debug:
      msg: "sequence is {{ item }}"
    with_sequence: start=1 end=5

I'm expecting the order 1,2,3,4,5  in order , but this prints the item in random fashion.

ok: [web01] => (item=3) => {
    "item": "3",
    "msg": "sequence is 3"
}
ok: [web01] => (item=2) => {
    "item": "2",
    "msg": "sequence is 2"
}
ok: [web01] => (item=1) => {
    "item": "1",
    "msg": "sequence is 1"
}
ok: [web01] => (item=5) => {
    "item": "5",
    "msg": "sequence is 5"
}
ok: [web01] => (item=4) => {
    "item": "4",
    "msg": "sequence is 4"
}

```yaml

```

##### EXPECTED RESULTS

expected order -  print  sequence from 1 to 5 in order

```

```

**Comment by nrwahl2** [comment]
I can't reproduce. Any configuration changes or plugin file changes on your machine?

Your `msg` parameter should be indented but I don't see how that would be related to the issue you're facing.

```
[reid@laptop ~/git]$ ansible --version
ansible 2.3.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.13 (default, Sep  5 2017, 08:53:59) [GCC 7.1.1 20170622 (Red Hat 7.1.1-3)]
[reid@laptop ~/git]$ cat test.yml
---
- hosts: all
  connection: local
  tasks:
  - name: sequence check
    debug:
      msg: "sequence is {{ item }}"
    with_sequence: start=1 end=5

[reid@laptop ~/git]$ ansible-playbook test.yml -i hosts.ini

PLAY [all] ********************************************************************************************************************************************************************************************************

TASK [Gathering Facts] ********************************************************************************************************************************************************************************************
ok: [localhost]

TASK [sequence check] *********************************************************************************************************************************************************************************************
ok: [localhost] => (item=1) => {
    "item": "1", 
    "msg": "sequence is 1"
}
ok: [localhost] => (item=2) => {
    "item": "2", 
    "msg": "sequence is 2"
}
ok: [localhost] => (item=3) => {
    "item": "3", 
    "msg": "sequence is 3"
}
ok: [localhost] => (item=4) => {
    "item": "4", 
    "msg": "sequence is 4"
}
ok: [localhost] => (item=5) => {
    "item": "5", 
    "msg": "sequence is 5"
}

PLAY RECAP ********************************************************************************************************************************************************************************************************
localhost                  : ok=2    changed=0    unreachable=0    failed=0  
```

**Comment by tharanga-abeyseela** [comment]
following is my config

[root@localhost ansible-2]# ansible-playbook -i web01, loop.yaml

PLAY [all] ****************************************************************************************************************************

TASK [Gathering Facts] ****************************************************************************************************************
ok: [web01]

TASK [checking loop] ******************************************************************************************************************
ok: [web01] => (item=3) => {
    "item": "3",
    "msg": "sequence is 3"
}
ok: [web01] => (item=2) => {
    "item": "2",
    "msg": "sequence is 2"
}
ok: [web01] => (item=1) => {
    "item": "1",
    "msg": "sequence is 1"
}
ok: [web01] => (item=5) => {
    "item": "5",
    "msg": "sequence is 5"
}
ok: [web01] => (item=4) => {
    "item": "4",
    "msg": "sequence is 4"
}

PLAY RECAP ****************************************************************************************************************************
web01                      : ok=2    changed=0    unreachable=0    failed=0

[root@localhost ansible-2]# ansible-playbook --version
ansible-playbook 2.3.2.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
  python version = 2.7.5 (default, Nov  6 2016, 00:28:07) [GCC 4.8.5 20150623 (Red Hat 4.8.5-11)]

**Comment by tharanga-abeyseela** [comment]
[root@localhost ansible-2]# ansible-playbook --version
	ansible-playbook 2.3.2.0
	  config file = /etc/ansible/ansible.cfg
	  configured module search path = Default w/o overrides
	  python version = 2.7.5 (default, Nov  6 2016, 00:28:07) [GCC 4.8.5 20150623 (Red Hat 4.8.5-11)]
	[root@localhost ansible-2]# cat loop.yaml
	---
	- hosts: all
	  connection: local
	  tasks:
	  - name: checking loop
		debug:
		  msg: "sequence is {{ item }}"
		with_sequence: start=1 end=5

	[root@localhost ansible-2]# clear
	[root@localhost ansible-2]# ansible-playbook --version
	ansible-playbook 2.3.2.0
	  config file = /etc/ansible/ansible.cfg
	  configured module search path = Default w/o overrides
	  python version = 2.7.5 (default, Nov  6 2016, 00:28:07) [GCC 4.8.5 20150623 (Red Hat 4.8.5-11)]
	[root@localhost ansible-2]# cat loop.yaml
	---
	- hosts: all
	  connection: local
	  tasks:
	  - name: checking loop
		debug:
		  msg: "sequence is {{ item }}"
		with_sequence: start=1 end=5

	[root@localhost ansible-2]# ansible-playbook -i web01, loop.yaml

	PLAY [all] ****************************************************************************************************************************

	TASK [Gathering Facts] ****************************************************************************************************************
	ok: [web01]

	TASK [checking loop] ******************************************************************************************************************
	ok: [web01] => (item=5) => {
		"item": "5",
		"msg": "sequence is 5"
	}
	ok: [web01] => (item=4) => {
		"item": "4",
		"msg": "sequence is 4"
	}
	ok: [web01] => (item=3) => {
		"item": "3",
		"msg": "sequence is 3"
	}
	ok: [web01] => (item=2) => {
		"item": "2",
		"msg": "sequence is 2"
	}
	ok: [web01] => (item=1) => {
		"item": "1",
		"msg": "sequence is 1"
	}

	PLAY RECAP ****************************************************************************************************************************
	web01                      : ok=2    changed=0    unreachable=0    failed=0

**Comment by bcoca** [comment]
duplicate of #21008, it is not the loop order, it is the 'result display order' that is sometimes affected when the machine is loaded, fixed by #28111
